utils/memset_secure.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __STDC_LIB_EXT1__
+#define __STDC_LIB_EXT1__
+#endif
```

Comment:
I'm not sure you should do that, can you provide a link where you saw this ?

Reply:
I got it from https://en.cppreference.com/w/c/string/byte/memset, but I realized I misread this part 🤦‍♂️  : 

> As with all bounds-checked functions, memset_s is only guaranteed to be available if __STDC_LIB_EXT1__ is defined by the implementation and if the user defines __STDC_WANT_LIB_EXT1__ to the integer constant 1 before including string.h.

Will fix